model/task_lifecycle.go

```diff
@@ -768,6 +768,15 @@ func removeNextMergeTaskDependency(cq commitqueue.CommitQueue, currentIssue stri
 			Status: AllStatuses,
 		}
 		if err = nextMerge.AddDependency(d); err != nil {
+			if d.TaskId == "" {
```

Comment:
I'm not sure it really makes sense to log this here; it doesn't seem like AddDependency errors specifically for the empty ID case